test/functional/mempool_accept_v3.py

```diff
@@ -212,6 +212,23 @@ def test_nondefault_package_limits(self):
                 tx_v3_child_large2["hex"])
         self.check_mempool([tx_v3_parent_large2["txid"]])
 
+    @cleanup
+    def test_fee_dependency_replacements(self):
+        """
+        Since v3 introduces the possibility of 0-fee (i.e. below min relay feerate) transactions in
+        the mempool, it's possible for these transactions' sponsors to disappear due to RBF.
+        Document that the 0-fee transaction should be evicted along with the replacements.
```

Comment:
I don't think this case is actually covered

rebased branch is failing on this test: https://github.com/bitcoin/bitcoin/pull/26403/files#diff-d18bbdec91d0f4825b512a31f34666b000c7b7e2e05a3d43570c4b971532616fR409